draft-ietf-quic-ack-frequency.md

```diff
@@ -121,18 +121,17 @@ This document uses terms, definitions, and notational conventions described in
 # Motivation
 
 A receiver acknowledges received packets, but it can delay sending these
-acknowledgements. The delaying of acknowledgements can impact connection
-throughput, loss detection and congestion controller performance at a data
-sender, and CPU utilization at both a data sender and a data receiver.
+acknowledgements. Delaying acknowledgements can impact connection
+throughput, loss detection and congestion controller performance at the
+sender, and CPU utilization at both the sender and receiver.
 
-Reducing the frequency of acknowledgement packets can improve connection and
+Reducing the frequency of acknowledgements can improve connection and
 endpoint performance in the following ways:
 
-- Sending UDP packets can be noticeably CPU intensive on some
-  platforms. Reducing the number of packets that only contain acknowledgements
-  can therefore reduce the amount of CPU consumed at a data receiver. Experience
-  shows that this cost reduction can be significant for high bandwidth
-  connections.
+- Sending UDP packets can be very CPU intensive on some platforms. Reducing
+  the number of ACK only packets reduces the CPU consumed at a data
```

Comment:
```suggestion
  the number of ACK-only packets reduces the CPU consumed at a data
```
Or maybe even "packets only carrying an acknowledgement frame"? Not sure we defined "ACK-only" anywhere?